.travis.yml

```diff
@@ -18,22 +18,24 @@ services:
 
 matrix:
   include:
-    - php: 5.3
+    - php: 5.4
       env: COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
You can move this section in the [allowed failures](https://docs.travis-ci.com/user/customizing-the-build#Rows-that-are-Allowed-to-Fail), `--prefer-lowest` install outdated versions and waste time during PRs.